package.json

```diff
@@ -38,7 +38,7 @@
     "grunt-jsbeautifier": "^0.2.12",
     "grunt-mocha-test": "^0.12.7",
     "grunt-nodemon": "^0.4.2",
-    "grunt-if": "^0.1.5",
+    "grunt-if": "https://github.com/binarymist/grunt-if/tarball/master",
```

Comment:
This addresses https://github.com/OWASP/NodeGoat/issues/77
The changes to grunt-if are here: https://github.com/binarymist/grunt-if/commit/01131e5c43c0c4198aa785192416098b1ba52a5e

It's basically dependencies. The maintainer is unresponsive, so the package on npm is no longer maintained. I simply forked and did what the package maintainer should have.